server/executor/linter_runner.go

```diff
@@ -8,7 +8,7 @@ import (
 
 	"github.com/kubeshop/tracetest/server/analytics"
 	"github.com/kubeshop/tracetest/server/linter"
-	linterResource "github.com/kubeshop/tracetest/server/linter/resource"
+	analyzerResource "github.com/kubeshop/tracetest/server/linter/analyzer"
```

Comment:
should this be just `analyzer`? is there a reason for the alias?

Reply:
Good catch! I'll update it.